lib/gunit/msvc15/gtest/gtest.vcproj

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="Windows-1252"?>
```

Comment:
The original project files did not have 32 bit targets. Hence I manually created them in the new project files..